dev/devicelab/bin/tasks/run_release_test_macos.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter_devicelab/common.dart';
+import 'package:flutter_devicelab/framework/devices.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/task_result.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+import 'package:path/path.dart' as path;
+
+/// Basic launch test for desktop operating systems.
+void main() {
+  task(() async {
+    deviceOperatingSystem = DeviceOperatingSystem.macos;
+    final Device device = await devices.workingDevice;
+    final Directory appDir = dir(path.join(flutterDirectory.path, 'dev/integration_tests/ui'));
+    await inDirectory(appDir, () async {
+      final Completer<void> ready = Completer<void>();
+      final List<String> stdout = <String>[];
+      final List<String> stderr = <String>[];
+
+      print('run: starting...');
+      final Process run = await startProcess(
```

Comment:
Use `startFlutter` instead, it handles `--local-engine`.  You may need update it to pipe `isBot` through that though.
https://github.com/flutter/flutter/blob/9072a09958a3fd9eaf4a857acbcc4b4e72167bbd/dev/devicelab/lib/tasks/microbenchmarks.dart#L39

Reply:
Done.